.evergreen/config.yml

```diff
@@ -2323,6 +2346,12 @@ buildvariants:
     tasks:
       - name: ".kms-kmip"
 
+  - matrix_name: "fuzz-test"
+    matrix_spec: { version: ["latest"], os-ssl-40: ["ubuntu1804-64-go-1-18"] }
```

Comment:
Similar to GODRIVER-2548, I'm wondering if we should use a stable version of the server in our fuzz testing (like 5.0 as of right now). At the moment, we're only testing the BSON library, which is hypothetically orthogonal to the server version, but if we start fuzzing the mongo libraries, I think it might be prudent to keep the server version stable so that we can be sure new fuzz errors are due to driver bugs.

Reply:
I think that makes sense, since this required building a matrix should we just test both?